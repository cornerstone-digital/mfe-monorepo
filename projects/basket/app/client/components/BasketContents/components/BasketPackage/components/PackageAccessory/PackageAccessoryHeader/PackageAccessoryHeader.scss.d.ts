export type Styles = {
  'package-header': string
}

export type ClassNames = keyof Styles

declare const styles: Styles

export default styles
