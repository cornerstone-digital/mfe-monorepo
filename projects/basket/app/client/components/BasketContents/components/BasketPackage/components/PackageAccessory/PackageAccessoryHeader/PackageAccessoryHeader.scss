.package-header {
  padding: $spacing-fixed-1 $spacing-fixed-2;
  border-bottom: 1px solid $color-pale-grey;
}
