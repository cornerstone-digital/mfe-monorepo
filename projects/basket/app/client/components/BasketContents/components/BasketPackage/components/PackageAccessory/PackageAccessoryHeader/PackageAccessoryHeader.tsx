import Heading from '@web-core/components/atoms/Heading'

import styles from './PackageAccessoryHeader.scss'

const PackageAccessoryHeader = () => {
  return (
    <div className={styles['package-header']}>
      <Heading level={5} size={6} margin={0} fontWeight="bold" appearance="inherit">
        Accessories
      </Heading>
    </div>
  )
}

export default PackageAccessoryHeader
