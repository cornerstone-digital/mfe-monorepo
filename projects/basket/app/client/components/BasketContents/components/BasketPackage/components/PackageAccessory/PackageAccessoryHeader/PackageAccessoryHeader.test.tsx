import { render } from '@testing-library/react'
import PackageAccessoryHeader from './PackageAccessoryHeader'

describe('PackageAccessoryHeader', () => {
  it('should render for basic scenario', () => {
    const { getByText, queryByText, asFragment, getByRole } = render(<PackageAccessoryHeader />)

    expect(getByRole('heading').textContent).toBe('Accessories')
    expect(getByText('Accessories')).toBeInTheDocument()
    expect(queryByText('Remove')).not.toBeInTheDocument()
    expect(asFragment()).toMatchSnapshot()
  })
})
