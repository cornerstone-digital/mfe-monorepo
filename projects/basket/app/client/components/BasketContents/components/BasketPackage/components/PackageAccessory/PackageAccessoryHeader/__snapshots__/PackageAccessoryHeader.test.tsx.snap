// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PackageAccessoryHeader should render for basic scenario 1`] = `
<DocumentFragment>
  <div
    class="package-header"
  >
    <h5
      class="heading heading-6 inherit w-bold margin-0"
    >
      Accessories
    </h5>
  </div>
</DocumentFragment>
`;
